ext/reflection/php_reflection.stub.php

```diff
@@ -910,5 +910,7 @@ public function getValue(): mixed {}
 
     public function isDeprecated(): bool {}
 
+    public function getFileName(): string|false {}
```

Comment:
There is no test case for the path where `getFileName` returns `false`. It would be useful to add such.

Reply:
Its the first test case in `ReflectionConstant_getFileName.phpt` - the file name for `PHP_VERSION` is false because it is provided by PHP

Reply:
That's true, but what do you think about improving the test coverage? You modified many places, thus paths, but provided only 4 test cases.

There are explicit changes to magic constants, com_dotnet ext, constants of different types, etc.

Reply:
I think the implementation is pretty straightforward and that extra tests are not really needed

Reply:
Full path coverage is always worth it. I don't understand your point.

Reply:
I added tests for the other manually-registered constants, except for from the com_dotnet extension which I wasn't sure how to reach